lib/types.ts

```diff
@@ -161,6 +161,11 @@ export type LeaderboardPageProps = {
     ordering?: "asc" | "desc";
   };
 };
+export type ContributorsPageProps = {
+  searchParams: {
+    search?: string;
+  };
+};
```

Comment:
Let's keep the Page Props in the `page.tsx` itself instead as this wouldn't be reused. Only keep reusable types in this file